google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/AppEngineDeploy.java

```diff
@@ -97,16 +98,22 @@ public void deploy(
     if (environment.isStandard() || environment.isFlexCompat()) {
       appYamlName = FlexibleFacetEditor.APP_YAML_FILE_NAME;
     } else {
+      String moduleName = deploymentConfiguration.getModuleName();
```

Comment:
I think more importantly, this method has become extremely long. I get that it has to do a lot of things, but it could be broken down into smaller logical groups to make it easier to read and digest. Could you put a TODO to refactor this?